subscriptions/providers/apple_in_app/__init__.py

```diff
@@ -247,6 +253,7 @@ def _handle_receipt(self, request: Request, payload: AppleReceiptRequest) -> Res
         return Response(SubscriptionPaymentSerializer(latest_payment).data, status=HTTP_200_OK)
 
     def _handle_new_subscription(self, notification: AppStoreNotification) -> None:
+        logger.debug('New subscription add operation started.')
```

Comment:
More info? `logger.debug('Handling new subscription for notification %s', notification)` or so maybe?